### ISSUE TITLE
Quest on Synergi 

### ISSUE THREAD

**Issue by Irrenka** [GameFreedomGG/Sindragosa#126]
Warmage Anzim, Borean Tundra, my race dranei, paladin, 74lvl, 

When I submitted the previous Q Taking Wing, so another player handed him over with him, he got another Q, but I got it bugged and I don't have the next Q at the NPC. That's why it doesn't send me to another Caldara location.

Thanks for the help i really need it this Q.

**Comment by Raulfin** [comment]
He is only accepting 1 turn in every appx 5-7min.

**Comment by MrTide** [comment]
Yes this event goes on cooldown until a player finishes it. about 5min i don't think its bugged
